### PR TITLE
DOCS-2275 Lint README

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Lint README to follow documentation [contributing](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) guidelines

### Motivation
Docs hackday project

### Additional Notes
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,7 +332,7 @@ end
...
 | `logs`                     | The fields used to fill values under the the `logs` section in the integration configuration file.                                                                                                                                                                                             |
-| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but will default to `true` in a future major version of the cookbook. |
+| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but defaults to `true` in a future major version of the cookbook. |
```
Comment (kayayarai) [inline README.md:335]: I dunno, this instance is specifically talking about a future major version. "will default" is correct if we're going to keep that. Or maybe "might default"

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ The Datadog Chef recipes are used to deploy Datadog's components and configurati
 * Datadog Agent v6.x
 * Datadog Agent v5.x
 
-**Note**: This page may refer to features that are not available for your selected version. Check the README of the
+**Note**: This page may reference features that are not available for your selected version. Check the README of the
```
Comment (kayayarai) [inline README.md:9]: Not sure this is better than "refer to." How about "mention" or "discuss" instead.

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ruthnaebeck) [reply README.md:335]:
```suggestion
| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but may default to `true` in a future major version of the cookbook. |
```

File: README.md (hunk 2 above)
Comment (ruthnaebeck) [reply README.md:9]:
```suggestion
**Note**: This page may discuss features that are not available for your selected version. Check the README of the
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**